pkg/cli/image_cmd_test.go

```diff
@@ -187,6 +187,84 @@ func TestSearchImageCmd(t *testing.T) {
 	})
 }
 
+func TestListRepos(t *testing.T) {
```

Comment:
Also add a 'real test' as above, where you start a zot server, upload an image and check that zli repos returns it

Reply:
added at line 220